tests/sentry/feedback/test_feedback_ingest.py

```diff
@@ -178,14 +176,12 @@ def test_bad_slug_path(self):
     def test_missing_optional_fields(self):
         # Optional fields missing should still result in successful save
         test_data_missing_optional_fields = {
-            "environment": "production",
```

Comment:
Hmm I wonder how our error events are treated when no environment, e.g. is it allowed or if we set a default when ingesting

Reply:
I recall learning that if no environment is set it defaults to "prod" or "production" one of those but I could be misremembering